src/index.html

```diff
@@ -152,11 +159,6 @@ <h4 class="card-title">
               </div>
             </div>
           </div>
-        </div>
-      </section>
-
-      <section>
-        <div class="row d-flex justify-content-center">
```

Comment:
I found this improves a little bit the alignment, by not taking so much space: now Connect Actions and BAsic Actions will be in the same line

![Screenshot from 2024-05-03 12-39-10](https://github.com/MetaMask/test-dapp/assets/54408225/75aa22b5-007f-4130-ab69-286137404924)


before:

![Screenshot from 2024-05-03 12-40-11](https://github.com/MetaMask/test-dapp/assets/54408225/f7cffe5b-076d-4cf5-bbba-a3ead8ea7ccf)